src/debugger/debugger.c

```diff
@@ -437,6 +441,8 @@ void xdebug_debugger_error_cb(zend_string *error_filename, int error_lineno, int
 
 	/* Send notification with warning/notice/error information */
 	if (XG_DBG(context).send_notifications && !XG_DBG(context).inhibit_notifications) {
+		xdebug_rebuild_stack_if_disabled();
```

Comment:
If we are going to notify on error, we need to rebuild the stack before sending the notification